public/stylesheets/style.css

```diff
@@ -32,7 +32,9 @@ a {
 #team-selection{
   margin-bottom: 15px;
 }
+.tabs: {
```

Comment:
can we go over this today

On Tue, May 5, 2015 at 5:16 PM, Blake Parkinson <notifications@github.com>
wrote:

> In public/stylesheets/style.css
> <https://github.com/blakeparkinson/sports2/pull/325#discussion_r29726647>:
>
> > @@ -32,7 +32,9 @@ a {
> >  #team-selection{
> >    margin-bottom: 15px;
> >  }
> > +.tabs: {
>
> wat?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/325/files#r29726647>.
>



-- 
*Henry Lyford*
Manager, Partnerships & Business Development
Edmodo.com/henry